src/Overture/Preliminaries.lagda

```diff
@@ -58,6 +58,9 @@ data 𝟘 : Type ℓ₀ where  -- We could use ⊥ instead.
 data 𝟙 : Type ℓ₀ where
  𝟎 : 𝟙
 
+data 𝟙⁺{ι : Level} : Type ι where
+ 𝟎 : 𝟙⁺
```

Comment:
That's a very unusual name for an inhabitant of 1.  Plus, `Unit.Polymorphic` exists in stdlib.

Reply:
Okay, point taken.  I replaced this and the empty type with \top and \bot from the std lib.

I'm okay with these, but they do look a bit out of place in examples where the two and three element types also appear.  Then we have:
```agda
⊥, ⊤, 𝟚, 𝟛
```

Reply:
It's fine to create synonyms for them, rather than redefine them entirely.